DrinkingPub/Order.cs

```diff
@@ -0,0 +1,23 @@
+﻿
+
+namespace Vsite.Oom.DrinkingPub
+{
+    public class Order
+    {
+        public Dictionary<string, int> Items = [];
```

Comment:
`Items` bi trebalo enkapsulirati tako da se ne može nekontrolirano mijenjati iz drugih klasa:
1. Staviti da Items ima private pravo pristupa
2. Definirati svojstvo `public IEnumerable<KeyValuePair<string, int>> OrderItems => Items.AsEnumerable();` koje 'e omogućiti čitanje stavki iz narudžbe.

Reply:
Dali je u redu ako sam koristio IReadOnlyDictionary umjesto IEnumerable kako bih i dalje mogao dohvaćati po indexu?